docs/source/deltacode/comprehensive_installation.rst

```diff
@@ -1,14 +1,14 @@
 Comprehensive Installation
 ==========================
 
-DeltaCode requires Python 3.6+ and is tested on Linux, Mac, and Windows. Make sure Python 3.6 (or above) is
-installed first.
+DeltaCode requires Python 3.6+ and is tested on Linux, Mac, and Windows. Make sure
+Python 3.6 (or above) is installed first.
 
 System Requirements
 -------------------
 
-- Hardware : DeltaCode will run best with a modern X86 processor and at least 1GB of RAM and 250MB
-  of disk.
+- Hardware : DeltaCode will run best with a modern X86 processor and at least 1GB of RAM and
+  250MB of disk.
```

Comment:
We need to revisit deltacode documentation once to check everything is up to date and consistant, let's create an issue for that (or after we merge in scancode-toolkit).

Reply:
Created issue: https://github.com/nexB/deltacode/issues/188
Will pick this up when we finish merging it with sc-tk.